app/javascript/services/alerts.js

```diff
@@ -1,13 +1,14 @@
 import axios from 'axios';
-import moment from 'moment';
+// import moment from 'moment';
 
 const REQUEST_URL = `${process.env.GFW_API_HOST_PROD}`;
-const DATASET = process.env.GLAD_PRECALC_DATASET;
+const ISO_DATASET = process.env.ISO_GLAD_SUMMARY;
+const ADM1_DATASET = process.env.ADM1_GLAD_SUMMARY;
```

Comment:
Can you names these by GLAD_.... instead? So they are grouped together in the alpha in heroku so easier to find.